Preprocessing/Makefile

```diff
@@ -2,4 +2,4 @@ install:
 	poetry install
 
```

Comment:
@sbancal maybe a good idea to add the tst_data run command to the Makefile ? Or is it on purpose ?

Reply:
don't understand, sorry ...

Reply:
@guilbep proposed that you add a command to the makefile that allows to test a specific dataset, is that possible? Doesn't seem like a priority to me - except if you wan to call this make command fromthe "Data checker" tab later :)

Reply:
Ok I understand.
As far as I know, Makefile calls like `make something` is not really meant to be called with arguments.
One workaround could be https://stackoverflow.com/a/2214593/446302 ... but it's not really easier than to call the `tst_preprocessing.py --help` to know how to run it with a specific dataset.

In the perspective to run it on dataset received from the data checker web, I plan to run the Python thing without the Makefile.

I see the Makefile more like the easy entry point for the sysadmin or end user.

Reply:
@sbancal  my point was more in the line: What's the purpose of this script ? and how and when should I use it ? And the Makefile is kind of a 'documentation' on how we should use the scripts: Is it an install script ? a test script ? a build script ?

Reply:
The purpose of this Makefile is to provide a `make install` and `make run` to help the sysadmin/end-user to run these common commands.
In the case one want to run it with more parameters, it becomes a documentation ... and help to know how to launch `tst_preprocessing.py` by hand

By the way, it is not a *test* script. *tst* has the meaning documented here https://github.com/EPFL-ENAC/CCFatiguePlatform#app-architecture (top left of the image)
It's a *preprocessing* script for *TST* files.